tests/zfs-tests/tests/functional/mmp/mmp.kshlib

```diff
@@ -163,17 +163,29 @@ function mmp_pool_set_hostid # pool hostid
 	return 0
 }
 
+function activity_check_duration
```

Comment:
Can you comment this function's return value, or give it a more verbose name like `total_seconds_mmp_has_been_active`?

Reply:
Done